core/chains/evm/config/config.go

```diff
@@ -158,6 +160,33 @@ func (c *chainScopedConfig) validate() (err error) {
 		err = multierr.Combine(err, ocrerr)
 	}
 
+	chainType := c.ChainType()
+	if !chainType.IsValid() {
+		err = multierr.Combine(err, errors.Errorf("CHAIN_TYPE %q unrecognised", chainType))
+	} else if c.knownID && c.defaultSet.chainType != chainType {
+		err = multierr.Combine(err, errors.Errorf("CHAIN_TYPE %q cannot be used with chain ID %d", chainType, c.ChainID()))
+	} else {
+		switch chainType {
+		case chains.Arbitrum:
+			if gasEst := c.GasEstimatorMode(); gasEst != "FixedPrice" {
+				err = multierr.Combine(err, errors.Errorf("GAS_ESTIMATOR_MODE %q is not allowed with chain type %q - "+
```

Comment:
Why does the case match Arbitrum but the message includes optimism?

Reply:
:facepalm: copy/paste - fixed.